### ISSUE TITLE
Python 2.6 unit test failure

### ISSUE THREAD

**Issue by sivel** [ansible/ansible#11441]
The test appears to be passing in python 2.7, but failing in 2.6:

```
======================================================================
ERROR: test_error_conditions (units.parsing.yaml.test_loader.TestAnsibleLoaderBasic)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ansible/ansible/test/units/parsing/yaml/test_loader.py", line 126, in test_error_conditions
    self.assertRaises(loader.get_single_data)
  File "/home/travis/build/ansible/ansible/.tox/py26/lib/python2.6/site-packages/unittest2/case.py", line 748, in assertRaises
    return context.handle('assertRaises', args, kwargs)
  File "/home/travis/build/ansible/ansible/.tox/py26/lib/python2.6/site-packages/unittest2/case.py", line 174, in handle
    (name, self._base_type_str))
TypeError: assertRaises() arg 1 must be an exception type or tuple of exception types
```

The offending line is:

https://github.com/ansible/ansible/blob/devel/test/units/parsing/yaml/test_loader.py#L126

**Comment by sivel** [comment]
Resolved in #11443

**Comment by mzabuawala** [comment]
I am also getting similar error with Python3.6

```
File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/unittest/case.py", line 728, in assertRaises
    return context.handle('assertRaises', args, kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/unittest/case.py", line 158, in handle
    (name, self._base_type_str))
TypeError: assertRaises() arg 1 must be an exception type or tuple of exception types

```
